test/Makefile

```diff
@@ -17,6 +17,7 @@ test_targets += \
 	232c93d07b74-test \
 	35fa71a030ca-test \
 	500f9fbadef8-test \
+	4c519f1e2d9f-test \
```

Comment:
Let's rename it, it was always easier to reference tests with readable names.
"rw_merge_test" maybe?